download_data.py

```diff
@@ -30,7 +30,7 @@ def raw_files(video_id, assets, metadata):
                 continue  # highres_depth asset only available for video ids from upsampling dataset
```

Comment:
Please handle the `wide` and `wide_intrinsics` assets in this case as well. 
These 2 assets are only available for videos that are part of the upsampling dataset, so the download script should skip downloading them with a warning in this case.